collections/infrastructure/roles/sosreport/templates/sosreport.sh.j2

```diff
@@ -0,0 +1,167 @@
+#jinja2: lstrip_blocks: "True"
+#### Blue Banquise file ####
+## {{ ansible_managed }}
+
+#!/bin/bash
+
+#set -x
+
+if [ "$(arch)" != "x86_64" ]; then
+    echo "This script only supports x86_64 architecture at the moment."
+    exit 1
+fi
+
+JQ=$(type -p jq)
+SOS=$(type -p sos)
+SH1SUM=$(type -p sha1sum)
+PKGMGR=""
+
+
+tmp="{{ sosreport_tempfile_path }}"
+output_path="{{ sosreport_output_path }}"
+output_file="sosreport"
+
+function usage {
+    cat <<EOF
+Usage:
+  $0 [options]
+
+Note: the script must be run on the OMN node
```

Comment:
I would replace "the OMN node" with "a management node".

Reply:
yes indeed. Thanks.